utils.go

```diff
@@ -71,6 +66,11 @@ func defaults(root string) ([]Subsystem, error) {
 	if !isUserNS {
 		s = append(s, NewDevices(root))
 	}
+	// add the hugetlb cgroup if test of support doesn't error out
+	h, hErr := NewHugetlb(root)
+	if hErr == nil {
```

Comment:
I think you can maybe check for a `os.IsNotExist` error.  We can skip if it is an notexists error and return it if its anything else?